### PR TITLE
More chemicals part five

### PR DESCRIPTION
**Pull request by FlapstheMoth**
## About the PR
Rewrite of #138, orders, fixes, perfects some new chemicals.

## Chemicals
1. Bromine (reagent for future chemicals)
2. Magnesium (reagent for future chemicals)
3. Platinum (reagent for future chemicals)
4. Calcium (reagent for future chemicals)
5. Barium (reagent for future chemicals)
6. Helium (reagent for future chemicals)
7. Caffeine
8. Salicyclic Acid
9. Salbutamol

### CODE REVIEW

**Comment by FlapstheMoth** [comment]
this is glorpshit

**Review by Merrokitsune — CHANGES_REQUESTED** [review]
IMG_5983

File: Resources/Prototypes/Recipes/Reactions/medicine.yml [hunk 1]
```diff
@@ -672,3 +672,37 @@
       amount: 1
   products:
     SalineGlucose: 3
+
+- type: reaction
+  id: Salicyclic
```
Comment (Merrokitsune) [inline Resources/Prototypes/Recipes/Reactions/medicine.yml:677]: Either this needs to be changed.

File: Resources/Prototypes/Reagents/medicine.yml [hunk 2]
```diff
@@ -1426,10 +1426,76 @@
...
+            Airloss: -0.50
+            Brute: -0.50
       - !type:ChemVomit
         conditions:
           - !type:ReagentThreshold
-            min: 12.5
+            min: 10
+
+- type: reagent
+  id: SalicyclicAcid
```
Comment (Merrokitsune) [inline Resources/Prototypes/Reagents/medicine.yml:1438]: Or that should be changed.

File: Resources/Prototypes/Recipes/Reactions/medicine.yml (hunk 1 above)
Comment (FlapstheMoth) [reply Resources/Prototypes/Recipes/Reactions/medicine.yml]: it's definitely something wrong i did with this one, idk what's happening with it, i'll try to change the recipe and ID

**Comment by FlapstheMoth** [comment]
Done!

**Review by Merrokitsune — APPROVED** [review]
Looks good.

**Pull request merged**